appscale/tools/local_state.py

```diff
@@ -834,15 +854,16 @@ def get_oauth2_storage_location(cls, keyname):
 
 
   @classmethod
-  def cleanup_appscale_files(cls, keyname):
+  def cleanup_appscale_files(cls, keyname, remove_location=True):
```

Comment:
Is this named after locations.json? If so, please rename to "remove_locations" for consistency. Otherwise, `remove_info` could also be used.